exercises/quiz1.rs

```diff
@@ -13,7 +13,7 @@
 //
 // No hints this time ;)
 
-// I AM NOT DONE
```

Comment:
The code in this quiz is incorrect!

- [ ] Fix the code

Reply:
that's what happens when you rush and don't read the instructions at the beginning of the file.
fixed it..